package.json

```diff
@@ -23,16 +23,17 @@
   "homepage": "https://github.com/silvermine/eslint-config-silvermine#readme",
   "dependencies": {
     "@silvermine/eslint-plugin-silvermine": "git+https://github.com/silvermine/eslint-plugin-silvermine.git#9a6e324f951fe89d5b6b115a5881cd6ff14b5f96",
-    "@typescript-eslint/eslint-plugin": "2.13.0",
+    "@typescript-eslint/eslint-plugin": "2.16.0",
```

Comment:
Unrelated to this upgrade, but found while testing it with a TypeScript project:

`@typescript-eslint/eslint-plugin` has a peerDependency on `@typescript-eslint/parser`. Can you add a that dependency in this MR?

Reply:
Yup, thank you! Updated this PR with that addition. Also updated my PR comment with a link to relevant line that references the dependency.